.github/actions/on_device_tests/action.yaml

```diff
@@ -89,3 +92,61 @@ runs:
           --label author-${GITHUB_PR_HEAD_USER_LOGIN:-$GITHUB_COMMIT_AUTHOR_USERNAME} \
           --label author_id-${GITHUB_PR_HEAD_USER_ID:-$GITHUB_COMMIT_AUTHOR_EMAIL}
       shell: bash
+    - name: Download ${{ matrix.platform }} Test Results
+      if: always() && env.TEST_TYPE == 'unit_test'
+      env:
+        COBALT_XMLS_FILENAME: cobalt_xmls.zip
+        COBALT_ERROR_LOG: webDriverTestLog.ERROR
```

Comment:
Ideally we could leave that name behind on the lab side and bring something cleaner here, e.g stdout.log and stderr.log, but i know this would prompt several layers of touches. So lets just accept it for now.

Reply:
I renamed the files that we archive to get rid of the weird naming.